### PR TITLE
Feature/ses support bsc 1088787

### PR DESCRIPTION
**Pull request by dmpop**

### CODE REVIEW

File: xml/installation-installation-ses_integration.xml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+ <para>
+  The current version of &productname; supports integration with
+  &ses;. Integrating &ses; enables &ceph; block storage as well as object
+  and image storage services in &productname;.
+ </para>
+  <section xml:id="ses.installation">
+  <title>Enabling &ses; Integration</title>
+  <para>
+   The &ses; integration is provided through the
+   <package>ardana-extensions-ses</package> RPM package. As this package is
```
Comment (stefannica) [inline xml/installation-installation-ses_integration.xml]: I remembered we changed the package name to `ardana-ses`, to avoid using the extension term.

Comment (dmpop) [reply xml/installation-installation-ses_integration.xml]: Fixed. Thanks!

File: xml/installation-installation-ses_integration.xml [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+ <para>
+  The current version of &productname; supports integration with
+  &ses;. Integrating &ses; enables &ceph; block storage as well as object
+  and image storage services in &productname;.
+ </para>
+  <section xml:id="ses.installation">
+  <title>Enabling &ses; Integration</title>
+  <para>
+   The &ses; integration is provided through the
+   <package>ses-ardana</package> RPM package. As this package is
```
Comment (stefannica) [inline xml/installation-installation-ses_integration.xml]: It's `ardana-ses` :-)

Comment (dmpop) [reply xml/installation-installation-ses_integration.xml]: Oops, my bad! Fixed (again) :-)

**Pull request merged**